iocell/src/main/scala/chisel/IOCell.scala

```diff
@@ -63,76 +63,68 @@ class DigitalInIOCellBundle extends Bundle {
   val ie = Input(Bool())
 }
 
-abstract class IOCell extends BlackBox
+trait IOCell extends BaseModule
 
-abstract class AnalogIOCell extends IOCell {
-  val io: AnalogIOCellBundle
```

Comment:
I don't think you should have changed this; it was done this way intentionally to allow the user to extend the IOCellBundle with foundry IO cell pins. Can you have multiple `IO`s in `BlackBox`es now?

Reply:
I can add this feature back in

Reply:
Cool, thanks. I think everything else looks good, for the record.

Reply:
Actually, it seems like you can do this in the current system: 
```
class FancyDigitalGPIOCellBundle extends DigitalGPIOCellBundle {
  val extra = Output(Bool())
}

class FancyDigitalGPIOCell extends GenericIOCell with DigitalGPIOCell {
  override val io = IO(new FancyDigitalGPIOCellBundle)
}
```
so no changes are necessary?

Reply:
I don't think `override` works- you'll call `IO` twice and end up with phantom `IO`s that are disconnected (also I don't think you can call `IO` twice in `BlackBox` anyway)

Reply:
NVM the above doesn't work.